client/src/vtk/viewManager.js

```diff
@@ -61,7 +61,7 @@ function getView(proxyManager, viewType) {
       .forEach((s) => proxyManager.getRepresentation(s, view));
 
     // Update orientation
-    const { axis, orientation, viewUp } = VIEW_ORIENTATIONS[name];
+    const { axis, orientation, viewUp } = VIEW_ORIENTATIONS.LPS[name];
```

Comment:
Why is this hardcoded as LPS?

Reply:
This is simply in the constructor as a default so that we can create the view objects in the store upon initial load (before `currentProject` is guaranteed to be defined). Once the scans are loaded in (at which point `currentProject` IS guaranteed to be defined), the `VtkViewer` initializes its camera referencing the project's chosen orientation.

Reply:
Ok, could we write that in a comment here?